stories/organisms/components/filePicker/FilePicker.stories.mdx

```diff
@@ -97,6 +97,36 @@ It is a combination of the [FileInput](/docs/atoms-fileinput--overview) and the
 
 <ArgsTable story="Overview" />
 
+## Clear all
+
+Need to delete all uploaded files at once?  
```

Comment:
Not correct for me. There is no upload as such here, and I wouldn't use the word `delete` which might suggest that we are actually deleting files when this is not the case (luckily). Please preserve the lexical field in which the component fits. If it is a `FilePicker` (pick = select carefully from a group), then the description must include the terms: pick, choose, select, leave, deselect, throw out, etc... It is essential to have an accurate description for everyone's understanding.

Reply:
@ccamel 
My bad, this is a sentence I added myself in a suggestion.. 
You're absolutely right and we need to strictly conform the documentation to the behavior of the component.

Reply:
Hey! @fredericvilcot. No worries. 😌 

Yes we all know it's hard to get it right. Keep fighting and struggling altogether, you are a great team that leads to success. So take heart! ❤️

Reply:
@ccamel I will say:

```suggestion
Need to clear all selected files at once?  
```

Reply:
Good to me!